tests/nim.cfg

```diff
@@ -1,3 +1,10 @@
 --path:"../"
 --gc:arc
 --threads:on
+--cc:clang
+--define:useMalloc
+@if not windows:
```

Comment:
Turns out clang has no tsan support and I do not know where else windows could have it.

Given that Linux and macos should work similarly enough I'm prone to leave it with data-races for those two.